ARCs/arc-0200.md

```diff
@@ -0,0 +1,232 @@
+          "name": "to_",
+          "desc": "The destination of the transfer"
+        },
+        {
+          "type": "uint256",
+          "name": "value_",
+          "desc": "Amount of tokens to transfer"
+        }
+      ],
+      "returns": { "type": "bool", "desc": "Success" }
+    },
+    {
+      "name": "transferFrom",
+      "desc": "Transfers tokens from source to destination as approved spender",
+      "readonly": false,
+      "args": [
+        {
+          "type": "address",
+          "name": "from_",
+          "desc": "The source  of the transfer"
+        },
+        {
+          "type": "address",
+          "name": "to_",
+          "desc": "The destination of the transfer"
+        },
+        {
+          "type": "uint256",
+          "name": "value_",
+          "desc": "Amount of tokens to transfer"
+        }
+      ],
+      "returns": { "type": "bool", "desc": "Success" }
+    },
+    {
+      "name": "approve",
+      "desc": "Approve spender for a token",
+      "readonly": false,
+      "args": [
+        { "type": "address", "name": "spender_" },
+        { "type": "uint256", "name": "value_" }
+      ],
+      "returns": { "type": "bool", "desc": "Success" }
+    },
+    {
+      "name": "allowance",
+      "desc": "Returns the current allowance of the spender of the tokens of the owner",
+      "readonly": true,
+      "args": [
+        { "type": "address", "name": "owner_" },
+        { "type": "address", "name": "spender_" }
+      ],
+      "returns": { "type": "uint256", "desc": "The remaining allowance" }
+    }
+  ],
+  "events": [
+    {
+      "name": "Transfer",
+      "desc": "Transfer of tokens",
+      "args": [
+        {
+          "type": "address",
+          "name": "from_",
+          "desc": "The source of transfer of tokens"
+        },
+        {
+          "type": "address",
+          "name": "to_",
+          "desc": "The destination of transfer of tokens"
+        },
+        {
+          "type": "uint256",
+          "name": "value_",
+          "desc": "The amount of tokens transferred"
+        }
+      ]
+    },
+    {
+      "name": "Approval",
+      "desc": "Approval of tokens",
+      "args": [
+        {
+          "type": "address",
+          "name": "owner_",
+          "desc": "The owner of the tokens"
+        },
+        {
+          "type": "address",
+          "name": "spender_",
+          "desc": "The approved spender of tokens"
+        },
+        {
+          "type": "uint256",
+          "name": "value_",
+          "desc": "The amount of tokens approve"
+        }
+      ]
+    }
+  ]
+}
+```
+
+Ownership of a token by a zero address indicates that a token is out of circulation indefinitely, or otherwise burned or destroyed.
+
+The methods `transfer` and `transferFrom` method MUST error when the balance of `from_` is insufficient.
+The `transferFrom` method MUST error unless called by an approved spender as defined by an extension defined in this ARC.
```

Comment:
Why does this say "approved spender as defined by an extension defined in this ARC"? I think you're just referring to the `approve` system, right?

Reply:
Yes, the `approve` system allows for multiple operators to spend on the behalf of other account if approved. I supposed we can revise it as follows.

approved spender -> `spender`

This may be more understandable since it matches the usage in the surrounding language and ABI spec.

Reply:
Revision here
https://github.com/algorandfoundation/ARCs/pull/223/commits/a9174bc215e756fb12858db4c0657cc0b73d2527

Reworded as `spender` approved by `owner` instead of approved spender. 
Added note that `from` is `owner` in case of `transfer` method